google_guest_agent/oslogin.go

```diff
@@ -103,12 +103,22 @@ func (o *osloginMgr) set() error {
 		logger.Errorf("Error updating group.conf: %v.", err)
 	}
 
-	for _, svc := range []string{"ssh", "sshd", "nscd", "unscd", "systemd-logind", "cron", "crond"} {
+	for _, svc := range []string{"nscd", "unscd", "systemd-logind", "cron", "crond"} {
 		if err := restartService(svc); err != nil {
 			logger.Errorf("Error restarting service: %v.", err)
 		}
 	}
 
+	// SSH should be explicitly started if not running.
+	for _, svc := range []string{"ssh", "sshd"} {
+		if err := startService(svc, true); err != nil {
```

Comment:
here. true argument means 'restart it if already running'

Reply:
Oh I see, why have the restartService function then? Effectively startService now does both.

Reply:
they map to systemctl behaviors, restartService only restarts if already running.